habitat-lab/habitat/tasks/rearrange/sub_tasks/pick_task.py

```diff
@@ -121,7 +121,17 @@ def reset(self, episode: Episode, fetch_observations: bool = True):
         start_pos, start_rot = self._gen_start_pos(sim, episode, sel_idx)
 
         sim.robot.base_pos = start_pos
-        sim.robot.base_rot = start_rot
+        # If the robot is Stretch, we force the base to rotate torward the object
+        if "stretch" in self._sim.robot.urdf_path:
```

Comment:
I think it's better to replace this with `isinstance(self._sim.robot, StretchRobot)`